Fix: end to end test bugs

jniles opened the pull request:
This PR fixes two end to end test bugs:

 1. The end to end tests have been updated for 2021.
 2. The employee config test correctly checks if the `bhCheckboxTree` component is active before blindly clicking on it.

It also includes one cosmetic change to `CHROME_OPTIONS` allowing devs to pass in multiple options by separating them with spaces as so:
```sh
CHROME_OPTIONS="--headless --disable-extensions"
```

Previous PR #5344

jniles committed:
fix(protractor): normalize chrome options

This change normalizes the CHROME_OPTIONS environmental variable so that
we can pass multiple options into the CHROME_BIN.  Basically, you can
now set your ~/.bashrc to have:

CHROME_OPTIONS="--headless --disable-gpu --no-dev-shm"

and everything will work correctly.

jniles committed:
fix(employees): fix employee config end to end test

Fixes the employee end to end tests.

jniles committed:
fix(fiscal): use correct FYs in E2E tests

Updates the fiscal years for 2021.

jmcameron commented on protractor.conf.js:
```diff
@@ -80,7 +80,10 @@ if (process.env.CI) {
-  config.capabilities.chromeOptions.args.push(process.env.CHROME_OPTIONS);
+
+  // normalize options so that multiple options can be used.
+  const opts = process.env.CHROME_OPTIONS.split(' ');
```
Nice!

jmcameron reviewed (commented):
The fixes look okay to me, but I did run into a problem when I tried the end-to-end tests on my laptop:

![image](https://user-images.githubusercontent.com/62145/106772557-fdd5b980-65f4-11eb-9aa2-8d2f586cbd66.png)

Not sure if this is related or not.  I have not run end-to-end tests on my system for a while so this could be a local issue.

dependabot-preview[bot] committed:
chore(deps): bump @types/angular from 1.8.0 to 1.8.1

Bumps [@types/angular](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/angular) from 1.8.0 to 1.8.1.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/angular)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

mbayopanda committed:
update puppeteer dependencies

mbayopanda committed:
Add mother name for patient in document

jmcameron reviewed (commented):
I ran all 4 tests on my laptop. units and integration pass.  E2E failed when trying to create a new FY.  Note that this is further than when I run it directly on the refactor branch or main (on my PC).
